src/main/java/ml/shifu/shifu/container/obj/ModelTrainConf.java

```diff
@@ -37,12 +37,12 @@
 
     /**
      * Different training algorithms supported in Shifu. SVM actuall is not implemented well. DT is replaced by RF and
-     * GBT.
+     * GBT. TF_DNN is used for tensorflow dnn training, TENSORFLOW is used for generic tensorflow model evaluation.
```

Comment:
what' the difference between TF_DNN and TENSORFLOW?

Reply:
Tensorflow is for the generic model evaluation. I'll delete it. It has been moved into generic model config